tsconfig.json

```diff
@@ -1,17 +1,37 @@
 {
   "compilerOptions": {
-    "target": "es2019",
-    "module": "esnext",
-    "moduleResolution": "Node",
-    "strict": true,
-    "jsx": "preserve",
-    "importHelpers": true,
-    "allowJs": true,
-    "skipLibCheck": true,
     "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
+    "skipLibCheck": true,
+    "target": "es2021",
+    "verbatimModuleSyntax": true,
+
+    "allowJs": true,
     "resolveJsonModule": true,
-    "sourceMap": true,
+    "moduleDetection": "force",
+
+    "strict": true,
+    "noUncheckedIndexedAccess": true,
+
+    "moduleResolution": "Node",
+    "module": "ESNext",
+    "noEmit": true,
+
+    "importHelpers": true,
+    "lib": [
+      "es2021",
```

Comment:
suggestion: In the video, they use 'es2022' for the 'target.' On line 5, we're still using 'es2021.'

Why can't we use the Es2022 standard? What challenges do you see?

Reply:
I decided to keep this, so we can enforce "older" JS, since we don't use a code parser anymore, to make the code compatible with older browsers. I would recommend to keep this for now.